src/Microsoft.DotNet.Build.Tasks.net45/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "Newtonsoft.Json": "7.0.1",
-    "NuGet.Versioning": "3.4.0-beta-488",
+    "NuGet.Versioning": "3.5.0-rc-1220",
```

Comment:
Oh I see, sorry, I guess we just need to update corefx to a new version of buildtools then.